server/models/alumniSchema.js

```diff
@@ -23,17 +24,23 @@ const alumniSchema = new Schema(
     name: {type: String, required: true},
     email: {type: String, reguired: true, unique: true},
     gradYear: {type: Number, required: true},
-    location: {type: String, required: false},
-    profession: {type: String, required: false},
-    company: {type: String, required: false},
-    college: {type: String, required: false},
+    country: {type: String, enum: COUNTRIES, required: true},
```

Comment:
Added corresponding string entries for `companyName`, `collegeName`, and `jobTitleName` to avoid lookups when alumni records are fetched